### DISCUSSION TITLE
New release: "Pelle" 2024.11.1

### DISCUSSION THREAD

**Discussion by pschuele** [tine-groupware/tine#86] [Announcements]
We just released the new major version of #tinegroupware "Pelle" 2024.11.1 🎉

Check https://tine-groupware.de, https://hub.docker.com/r/tinegroupware/tine and https://github.com/tine-groupware/tine/releases/tag/2024.11.1 for the release notes and the downloads.

Please also follow us on #bluesky -> @tine-groupware.bsky.social

**Comment by lab-at-nohl** [comment]
Nice look, I like it. One minor detail: I always wondered which resolution you recommend for displays (what do you use)? I at least need to zoom the website to 150 % to have readable text (buttons could be smaller then)...

Thanks for including the patches for ownCloud apps btw. All in all it is really a office workers release. 

`php72 setup.php --update` died w/o notice (Found only error message "COLUMN tree_fileobjects.flysystem not found"). Rerunning with **PHP 8.2** resolved this issue. -- Not a bug, it is said in the release notes! --

**Reply by pschuele** [reply]
> One minor detail: I always wondered which resolution you recommend for displays (what do you use)? I at least need to zoom the website to 150 % to have readable text (buttons could be smaller then)...

I have 2560x1440px and use 100% (no-)zoom. but I can see your point, you need good eyes for tine with 100% and I heard that from some people, that they have difficulties reading the grids... there are still some ui-glitches with higher zooms, i.e. some edit dialogs don't looking very good or content that slides out of view. but we are working on that (and more responsiveness).
